### DISCUSSION TITLE
[@types/office-js] The Office.auth(. getAccessToken) seems to be missing.

### DISCUSSION THREAD

**Discussion by wilcotroost** [DefinitelyTyped/DefinitelyTyped#56813] [Issues with a @types package] [answered]
I would like to use the Office.auth.getAccessToken in my Angular typescript but it seems to be missing from the type definition. 

Could it be that I am missing something?

Thanks for your time!

[a comment by an automated account is not shown]

**Comment by wilcotroost** [comment]
If I examin the index.d.ts file I can find the getAccessToken definition. It is part of the Auth interface and mentions the Office.auth in the comments. How does the transition from Office.Auth.. to Office.auth.. work? And why is the Office.Auth also not recognized?

**Comment by AlexJerabek** [comment]
Adding @davidchesnut and @wandyezj to the thread.

**Comment by ElizabethSamuel-MSFT** [comment, marked as answer]
@wilcotroost Are you still experiencing this? The Auth interface and its members should be available in the release version.

- [Office.Auth](https://learn.microsoft.com/javascript/api/office/office.auth?view=common-js)
